stream_django/tests/test_app/tests/test_model_integration.py

```diff
@@ -74,9 +82,13 @@ def test_enrich_data_with_missing_reference(self):
         # make sure missing data is correctly tracked
         enriched_data = self.enricher.enrich_activities([activity])
         self.assertFalse(enriched_data[0].enriched)
-        self.assertIn('object', enriched_data[0].not_enriched_data)
-        self.assertDictContainsSubset(dict(object=activity['object']), enriched_data[0].not_enriched_data)
```

Comment:
`assertDictContainsSubset` is deprecated, so switched it to `set1.issubset(set2)` as per the recommendation of almighty stackoverflow 😬